rg_instructor_analytics/tasks.py

```diff
@@ -0,0 +1,24 @@
+"""
+Module for celery tasks.
+"""
+from django.conf import settings
+from django.core.mail import EmailMultiAlternatives
+from django.template.loader import render_to_string
+from django.utils.html import strip_tags
+from lms import CELERY_APP
```

Comment:
Please split imported blocks with the empty line.

Reply:
done

Reply:
Please improve tox config file to fix this error